### ISSUE TITLE
Always run language status animation, even with reduced motion mode

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#148232]
Showing small motions (in terms of both amount of screen moving and for how long) that convey information is allowed even under reduced motion mode, this is noted in the WCAG standards: https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html

> Success Criterion [2.3.3 Animation from Interactions](https://www.w3.org/TR/WCAG21/#animation-from-interactions) (Level AAA): [Motion animation](https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html#dfn-motion-animation) triggered by interaction can be disabled, unless the animation is [essential](https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html#dfn-essential) to the functionality or the information being conveyed.

> **Essential**: if removed, would fundamentally change the information or functionality of the content, and information and functionality cannot be achieved in another way that would conform

Thus if we do not use some other way of messaging the user, it is appropriate to invoke a small animation.

_Originally posted by @JacksonKearl in https://github.com/microsoft/vscode/issues/145950#issuecomment-1110056707_
